packages/ckbtc/candid/bitcoin.did

```diff
@@ -67,6 +67,9 @@ type fees = record {
   get_current_fee_percentiles_maximum : nat;
   send_transaction_base : nat;
   send_transaction_per_byte : nat;
+  get_block_headers_base : nat;
+  get_block_headers_cycles_per_ten_instructions : nat;
+  get_block_headers_maximum : nat;
```

Comment:
This is a breaking change.
I'm checking with the author of https://github.com/dfinity/bitcoin-canister/pull/309 is this is intentional.

Reply:
I was informed that the `fees` type is only used to configure the canister, which we wouldn't be doing. So this should be OK.